NAMESPACE

```diff
@@ -22,11 +22,14 @@ importFrom(DGEobj,getItem)
 importFrom(assertthat,assert_that)
```

Comment:
@agenius-mohammed-ali - Added a few comments . Please check and revert back if you have any questions